apps/web/src/app.tsx

```diff
@@ -73,8 +73,8 @@ function App() {
 
   return (
     <div id="app">
-      <div id="sidebar" style={{ zIndex: 200 }}>
-        <Sidebar
+      <div id="community-sidebar" style={{ zIndex: 200 }}>
```

Comment:
Shouldn't be ids naming same as component naming, so sidebar first?
sidebar-community, same goes for sidebar-members etc.?

Reply:
done in https://github.com/status-im/status-web/pull/370/commits/47459664ba6f2ff544047ac712bf9b2b630da467